File: packages/graph/__fixtures__/extractor-project/auth_test.py
import pytest


class TestAuthService:
    def test_reject_expired_tokens(self):
        """Expired tokens should be rejected with 401"""
        assert True

    def test_accept_valid_jwt(self):
        """Valid JWT tokens grant access"""
        assert True

    def test_lock_account_after_failed_attempts(self):
        assert True


@pytest.mark.parametrize("role", ["admin", "user", "guest"])
def test_role_based_access(role):
    """Each role should have appropriate permissions"""
    assert role in ["admin", "user", "guest"]


def test_password_hashing():
    assert True

File: packages/graph/__fixtures__/extractor-project/auth_test.rs
#[cfg(test)]
mod tests {
    use super::*;

    /// Expired tokens should be rejected with 401
    #[test]
    fn test_reject_expired_tokens() {
        assert!(true);
    }

    /// Valid JWT tokens grant access
    #[test]
    fn test_accept_valid_jwt() {
        assert!(true);
    }

    #[test]
    fn test_lock_account_after_failed_attempts() {
        assert!(true);
    }

    #[test]
    fn test_password_hashing() {
        assert!(true);
    }
}
